catalog/video/video-mpegts-transformer/src/main/java/org/codice/alliance/transformer/video/MpegTsInputTransformer.java

```diff
@@ -132,6 +144,110 @@ public Metacard transform(InputStream inputStream)
         return transform(inputStream, null);
     }
 
+    /**
+     *
+     * @param classification must be non-null
```

Comment:
No need to put the non-null comments on there since this is what is expected by default (see https://groups.google.com/d/msg/ddf-developers/g23oWEdZDIw/s0O-dzWYAAAJ). I also updated https://codice.atlassian.net/wiki/display/DDF/Code+Comment+Guidelines with this information.